### DISCUSSION TITLE
jsforce v3 is now GA!

### DISCUSSION THREAD

**Discussion by cristiand391** [jsforce/jsforce#1529] [Announcements]
Hey everyone!

After a long wait, we’re happy to announce that jsforce v3 is now generally available (GA)!

With this new release, we also announce that jsforce v1/v2 have reached “End of Life”. The v1/2 branches remain part of the jsforce repo but won’t receive further updates. Projects that depend on jsforce must migrate to v3 for the latest security patches and new features.

Features available only in jsforce v3:

* jsforce-node: a smaller distribution of jsforce for Node.js projects.
* Support for the Salesforce Bulk API 2.0.
* Improved handling of network errors.

Thank you to everyone who invested time in testing jsforce v3 during development. Your ideas, feedback, and bug reports were an important part of reaching this milestone.

Special thanks to @stomita for their help in refactoring jsforce v1/2 and adding new features.

Sincerely, 
Cristian and Shinichi

**Comment by cristiand391** [comment]
### Migration

v1 → v3:
https://github.com/jsforce/jsforce/blob/main/MIGRATING_V1-V3.md

v2 → v3:
https://github.com/jsforce/jsforce/blob/main/MIGRATING_V2-V3.md

v3 docs:
https://jsforce.github.io/document/

API reference:
https://jsforce.github.io/jsforce/

Is there something not covered in the guides? Feel free to create a new post:
https://github.com/jsforce/jsforce/discussions/categories/q-a

Found a bug? File a new GH issue:
https://github.com/jsforce/jsforce/issues/new

**Comment by pavitra-infocusp** [comment]
This is such a great news! All thanks to you @cristiand391
